test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -34,3 +34,7 @@ module.exports = ->
     @createFile "routes.coffee", routesFileContents, (err) =>
       return done.fail err if err
       done()
+
+
+  @Then /^the app doesn't error$/, (done) ->
```

Comment:
Haha, okay `done` doesn't exist here. But the empty body is weird here. Doesn't this step always pass, whether there are errors or not?